covid_xprize/validation/tests/test_scenario_generator.py

```diff
@@ -340,77 +340,115 @@ def _check_future(self, start_date_str, end_date_str, scenario_df, scenario_npis
 
     def test_generate_scenario_mind_the_gap_freeze(self):
         # Scenario = Freeze
-        start_date_str = "2021-01-01"
```

Comment:
Mind the gap scenarios are supposed to have a start date AFTER the last known date. Now setting start date 7 days from last known date.

Mind the gap:    I____LK    S____E
  Definitions:
  I = inception date = 2020-01-01 (earliest available data)
  LK = last known date (for each country) in the latest available data
  S = start date of the scenario
  E = end date of the scenario